### PR TITLE
Maker training page prototype

### PR DESCRIPTION
**Pull request by MatthewGalan**
Completed initial prototype of training modules page for makers.

image

### CODE REVIEW

File: client/src/pages/maker/training/AvailableModuleThumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ModuleThumbnail from "./ModuleThumbnail";
+import { Typography } from "@mui/material";
+
+interface AvailableModuleThumbnailProps {
+  title: string;
+  image: string;
+  estimatedDuration: number;
```
Comment (pjw9963) [inline client/src/pages/maker/training/AvailableModuleThumbnail.tsx:8]: is estimatedDuration something we need added to the backend?

File: client/src/pages/maker/training/CompletedModuleThumbnail.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+interface CompletedModuleThumbnailProps {
+  title: string;
+  image: string;
+  completionDate: string;
+}
+
+export default function CompletedModuleThumbnail({
+  title,
+  image,
+  completionDate,
```
Comment (pjw9963) [inline client/src/pages/maker/training/CompletedModuleThumbnail.tsx:15]: completionDate seems like another thing the backend doesn't track yet

**Review by pjw9963 — APPROVED** [review]
Good work. Saw a couple fields here that don't have a counter part in the backend (yet) so I just took note of those

**Comment by MatthewGalan** [comment]
@pjw9963 Thanks for the review. `estimatedDuration` and `completionDate` were just two things I was brainstorming that could be useful. We can talk to Mike tomorrow to see if he agrees.

Proceeding with merge b/c it's just a prototype anyways.

**Pull request merged**